base/range.jl

```diff
@@ -860,13 +860,19 @@ end
 
 median(r::Range{<:Real}) = mean(r)
 
-function in(x, r::Range)
+function in(x::Real, r::Range{<:Real})
+    n = step(r) == 0 ? 1 : round(Integer,(x-first(r))/step(r))+1
```

Comment:
> Given that the definitions are so close, I doubt it.

Even in the relatively short time I've been reviewing actively, IIRC I've seen such happen at least once :).

> Also it turns out it's harder than it seems, since their signatures are different: T doesn't need to be the same for both arguments with the Real method.

Not certain I follow? Would e.g. the following not work?
```julia
in(x::Real, r::Range{<:Real}) = _therecanbeonlyone(x, r)
in(x::T, r::Range{<:T}) where {T} = _therecanbeonlyone(x, r)
function _therecanbeonlyone(x, r)
    n = step(r) == 0 ? 1 : round(Integer, (x - first(r))/step(r)) + 1
    n >= 1 && n <= length(r) && r[n] == x
end
```
Best! :)